Helmsman/utils/esignet-standalone-2.0.0/esignet-mosipid2-plugin-values-2-0-0.yaml

```
@@ -0,0 +1,139 @@
+  - name: LOGGING_LEVEL_ORG_SPRINGFRAMEWORK_WEB_CLIENT
+    value: DEBUG
+  - name: MOSIP_ESIGNET_CAPTCHA_REQUIRED
+    value: ""
+  - name: MOSIP_ESIGNET_CAPTCHA_URL
+    value: http://captcha.captcha
+  - name: "MOSIP_ESIGNET_MISP_KEY"
+    valueFrom:
+      secretKeyRef:
+        name: esignet-misp-onboarder-key
+        key: mosip-esignet-misp-key
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)